transform/_procesador_omie.py

```
@@ -0,0 +1,939 @@
+            return df
+
+        # Verify required columns exist
+        required_cols = ['Fecha', 'Hora', 'granularity']
+        if not all(col in df.columns for col in required_cols):
+            raise ValueError(f"Required columns: {required_cols} not found in DataFrame.")
+
+        # Ensure Fecha is datetime
+        df['Fecha'] = pd.to_datetime(df['Fecha'])
+        
+        # Get timezone object
+        tz = pytz.timezone('Europe/Madrid')
```

Ruff (0.11.9)

Local variable `tz` is assigned to but never used

Remove assignment to unused variable `tz`

(F841)